src/simcore_service_deployment_agent/config_schema.py

```diff
@@ -47,7 +47,7 @@
         ),
         "docker_stack_recipe": T.Dict(
             {
-                "files": T.List(
+                T.Key("files"): T.List(
```

Comment:
ah, so what Matus works on has nothing to do with trafaret.
But if you wish to remove this (which I agree is not very easy to use). What we do in osparc-simcore is to use pydantic settings facilities. I can show you how this looks like.

Reply:
yes, I agree. It is very easy to upgrade and there are many advantages coming with it

Reply:
thanks